applied-spectroscopy-reviews.csl

```diff
@@ -5,7 +5,7 @@
     <id>http://www.zotero.org/styles/applied-spectroscopy-reviews</id>
     <link href="http://www.zotero.org/styles/applied-spectroscopy-reviews" rel="self"/>
     <link href="http://www.zotero.org/styles/molecular-and-cellular-proteomics" rel="template"/>
-    <link href="Applied Spectroscopy Reviews" rel="documentation"/>
+    <link href="http://www.tandfonline.com/action/authorSubmission?journalCode=laps20&page=instructions" rel="documentation"/>
```

Comment:
Fixed in https://github.com/citation-style-language/styles/blob/master/applied-spectroscopy-reviews.csl.

No soup for you!